docs/operator-manual/application.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       parameters:
       - name: "nginx-ingress.controller.service.annotations.external-dns\\.alpha\\.kubernetes\\.io/hostname"
         value: mydomain.example.com
+      - name: "ingress.annotations.kubernetes\\.io/tls-acme"
```

Comment:
why the double backslash?

Reply:
Honestly, I copied the double backslash from the example above. Looks like first backslash is required for `helm template --set` and second backslash is required to escape first slash in YAML